### PR TITLE
Clear unused file & Resubmit Pull Request

### PR DESCRIPTION
**Pull request by Xin128**
The following changes I have implemented:
1) Week 2 Minimum requirement: paragraphs; images; website links...
2) Week 3 Step 1 and Step 2: fetching
3) Clearing up the unused files with original websites

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -23,10 +23,10 @@
...
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
+     
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html;");
+        response.getWriter().println("Hello Xin!");
+    }
```
Comment (cisxingg) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: Indentations don't need to change?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,45 @@
...
+      <h1>Xin Hao</h1>
+      <p>Code a Hello World.</p>
+      <p>Who am I?  </p> 
+      <button onclick="addRandomGreeting()">Click to See!</button> 
       <div id="greeting-container"></div>
+      <button onclick="getHellofromFetchedPractice()">Hello from  outside World!!</button>
+      <div id="fetch-container"></div>
+      <p>I am currently a rising CS junior at Rice University and STEP Intern at Google! <br>
+      I love coding, thinking, cooking, reading and traveling. <br>
+      "Talk is Cheap; Show me the CODE."
```
Comment (cisxingg) [inline portfolio/src/main/webapp/index.html]: br at the end as well?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,45 @@
...
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Xin Hao</h1>
+      <p>Code a Hello World.</p>
+      <p>Who am I?  </p> 
+      <button onclick="addRandomGreeting()">Click to See!</button> 
       <div id="greeting-container"></div>
+      <button onclick="getHellofromFetchedPractice()">Hello from  outside World!!</button>
+      <div id="fetch-container"></div>
+      <p>I am currently a rising CS junior at Rice University and STEP Intern at Google! <br>
+      I love coding, thinking, cooking, reading and traveling. <br>
```
Comment (cisxingg) [inline portfolio/src/main/webapp/index.html]: Indentation of 2 spaces for new lines inside <p>: See recommended example here: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns (just look at the example, this section is to explain something else :))

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -2,17 +2,45 @@
...
+      "Talk is Cheap; Show me the CODE."
+      Dream; Love; Win.
+      </p>
+      <div id="greeting-container"></div>
+      <p>These are the projects I have worked on:</p>
+        <ul class="links">
+            <li><a href="https://github.com/ZhiGroup/pytorch_ehr">EHR_Pytorch</a></li>
```
Comment (cisxingg) [inline portfolio/src/main/webapp/index.html]: Usually we do indentation of 2 spaces:
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

Comment (cisxingg) [reply portfolio/src/main/webapp/index.html]: Same thing below as well :)

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -26,3 +26,16 @@ function addRandomGreeting() {
...
+ * The above code is organized to show each individual step, but we can use an
+ * ES6 feature called arrow functions to shorten the code. This function
+ * combines all of the above code into a single Promise chain. You can use
+ * whichever syntax makes the most sense to you.
+ */
+function getHellofromFetchedPractice() {
+  fetch('/data').then(response => response.text()).then((quote) => {
+    document.getElementById('fetch-container').innerText = quote;
+  });
+}
```
Comment (cisxingg) [inline portfolio/src/main/webapp/script.js]: Does this work?
Also the comments here are a little confusing, can you rephrase them?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (Xin128) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Got it!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (Xin128) [reply portfolio/src/main/webapp/index.html:24]: Got it! Is it just line 24 that needs to be changed? That's the only place that has a new line inside <p> </p> block~

Comment (Xin128) [reply portfolio/src/main/webapp/index.html]: Just changed line 23 - 26

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (Xin128) [reply portfolio/src/main/webapp/index.html]: Yes! Sorry I forgot...

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (Xin128) [reply portfolio/src/main/webapp/index.html]: Changed them all. I need to switch my tab size later...

File: portfolio/src/main/webapp/script.js (hunk 5 above)
Comment (Xin128) [reply portfolio/src/main/webapp/script.js]: Yes I think this method works fine on my portfolio. I have added some further comments there on the newest version

**Review by cisxingg — APPROVED** [review]
LG, please wait for Warfa's review.

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
```
Comment (warfa-google) [inline portfolio/src/main/webapp/script.js:18]: Lets change greeting to something else since you're no longer generating a greeting

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -17,7 +17,7 @@
...
   const greetings =
```
Comment (warfa-google) [inline portfolio/src/main/webapp/script.js:19]: Same here and below

**Review by warfa-google — APPROVED** [review]
Looks good other than small comments
